### ISSUE TITLE
Accessibillity: Missing for= and ids in Forms

### ISSUE THREAD

**Issue by candidexmedia** [authorizerdev/authorizer#337]
Hi! I don't have a current deployment of Authorizer up and running, but I did want to note some accessibility issues with the [demo](https://authorizer.dev/) Log In and Sign Up forms (which I assume are using the latest version of Authorizer).

![image](https://user-images.githubusercontent.com/95003834/226064332-afc042aa-95fe-40d4-9a0c-7537a866939f.png)

The forms are missing key attributes like #ids, `for=`. These are important for accessibility (see: [Labeling Controls](https://www.w3.org/WAI/tutorials/forms/labels/) and [Accessibility concerns > Labels](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input#accessibility_concerns)). It will also make creating style customization with CSS easier in the future.

For instance, the following markup for the name field:

```html
<div>
  <label class="sc-eCYdqJ isqRzX"> <span class="sc-gsnTZi bvppDV">*</span>Email</label>
  <input name="email" type="email" placeholder="eg. foo@bar.com" class="sc-jSMfEi dQhCkx" value="">
</div>
```

... could be:

```
<div>
  <label for="auth-form-name" class="sc-eCYdqJ isqRzX"> <span class="sc-gsnTZi bvppDV">*</span>Email</label>
  <input id="auth-form-name" name="email" type="email" placeholder="eg. foo@bar.com" class="sc-jSMfEi dQhCkx" value="">
</div>
```

This will allow the field to be in focus when a user clicks on the label, and a screen reader will read out the label when the user is focused on the form input ([source](https://stackoverflow.com/questions/12894169/what-is-the-html-for-attribute-in-label)).
